src/main/app/src/app/shared/location/location.service.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-FileCopyrightText: 2022 Lifely
```

Comment:
Dit mag niet licentie-technisch gezien. Je mag niet zomaar een licentie header aanpassen van bestaande code. Je moet altijd de originele partij blijven noemen. Zie: https://github.com/infonl/dimpact-zaakafhandelcomponent/blob/main/CONTRIBUTING.md

Reply:
Wat je dus moet doen volgens onze licentie is de licentie header veranderen naar:

```
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
 ```